### ISSUE TITLE
General issues with command output and formatting

### ISSUE THREAD

**Issue by kyle-blair** [forcedotcom/cli#2186]
### Summary

Several `sf` commands seem to truncate output based on terminal width. This is problematic for a number of reasons, but primarily because it hides necessary information. In at least one case an error message disappeared entirely when my terminal hit a certain width.

### Steps To Reproduce:

1. An easy way to reproduce this issue is to set the "packageDirectories[].default" attribute to false (works for a package repo, not sure otherwise).
```json
{
  "packageDirectories": [
    {
      "path": "force-app",
      "default": false
...
}
```
2. Now try running a package version create command and observe the output that includes an error.
```shell
sf package version create -c -a "v0.1.0.0" -n 0.1.0.0 -p my-test-package -w 60
```
3. Expand the size (width) of your terminal window and rerun the command from step 2.

### Expected result

Commands output necessary information regardless of terminal width.

### Actual result

Commands truncate the output based on terminal width. Once my terminal reached a certain width the error information disappeared entirely...

### System Information

```json
{
  "cliVersion": "@salesforce/cli/1.78.0",
  "architecture": "darwin-x64",
  "nodeVersion": "node-v18.16.0",
  "osVersion": "Darwin 21.6.0",
  "shell": "bash",
  "rootPath": "/Users/user/.local/share/sfdx/client/7.200.6-d8413f2/sf",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 2.1.9 (core)",
    "@oclif/plugin-commands 2.2.14 (core)",
    "@oclif/plugin-help 5.2.9 (core)",
    "@oclif/plugin-not-found 2.3.23 (core)",
    "@oclif/plugin-plugins 3.0.1 (core)",
    "@oclif/plugin-search 0.0.17 (core)",
    "@oclif/plugin-update 3.1.13 (core)",
    "@oclif/plugin-version 1.3.3 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.36 (core)",
    "@oclif/plugin-which 2.2.19 (core)",
    "@salesforce/cli 1.78.0 (core)",
    "apex 2.2.15 (core)",
    "auth 2.7.14 (core)",
    "data 2.3.14 (core)",
    "deploy-retrieve 1.8.18 (core)",
    "info 2.6.10 (core)",
    "limits 2.3.13 (core)",
    "login 1.2.8 (core)",
    "org 2.8.1 (core)",
    "packaging 1.16.11 (user)",
    "schema 2.3.10 (core)",
    "settings 1.4.6 (core)",
    "sobject 0.1.18 (core)",
    "source 2.10.7 (core)",
    "telemetry 2.2.0 (core)",
    "templates 55.4.13 (core)",
    "trust 2.4.14 (core)",
    "user 2.3.10 (core)"
  ]
}
```
### Additional information

Sample output (all output appears different than it was initially printed since I increased my terminal width after the command had finished):
```shell
$ sf package version create -c -a "v0.1.0.0" -n 0.1.0.0 -p my-package -w 60
Version create....
Error (1): The sfdx-project.json file must include one, and only one, default package directory (path). Because your sfdx-project.json file contains only one package directory,Version create.... done
```
After increasing my terminal width more:
```shell
$ sf package version create -c -a "v0.1.0.0" -n 0.1.0.0 -p my-package -w 60
Version create....
Error (1): The sfdx-project.json file must include one, and only one, default package directory (path). Because your sfdx-project.json file contains only one package directory, it must be the default. Remove the `"Version create.... done
```
And finally, after increasing my terminal width even more, the error message disappears from the output:
```shell
$ sf package version create -c -a "v0.1.0.0" -n 0.1.0.0 -p my-package -w 60
Version create....
Version create.... done
```

Note that I'm on an ultrawide monitor, although I didn't need to use the entire width of my screen before the error message disappeared. Approximate terminal dimensions where the error disappeared:
```shell
echo $(tput cols)x$(tput lines)
294x46
```

**Comment by daveespo** [comment]
Duplicate of #2048

**Comment by kyle-blair** [comment]
> Duplicate of #2048

I made a meager attempt at searching open issues at least. Not sure if that issue does/will cover the behavior where the error message disappears completely when the terminal is wide enough?

**Comment by mshanemc** [comment]
yeah, it *should* be fixed by #2048
